lib/ansible/modules/database/postgresql/postgresql_user.py

```diff
@@ -220,14 +223,22 @@
 else:
     postgresqldb_found = True
 
-from ansible.module_utils._text import to_bytes
 from ansible.module_utils.six import iteritems
 
+try:
+    from passlib.hash import postgres_md5 as pm
+except ImportError:
+    passlib_hash_found = False
+else:
+    passlib_hash_found = True
```

Comment:
Just a couple style note, usually we import from stdlib, then from 3rd party libs, and then from ansible libs.

For setting whether passlib is found, you can do it like this to save a line:
``` python
try:
    from passlib.hash import postgres_md5 as pm
    passlib_hash_found = True
except ImportError:
    passlib_hash_found = False
```

Since passlib_hash_found is a toplevel constant, style-wise it should be all uppercase:
PASSLIB_HASH_FOUND